static/sass/styles.scss

```diff
@@ -1,5 +1,4 @@
-$color-tabs-active-bar: #e95420;
-
+$color-tabs-active-bar: #000;
```

Comment:
I don't think this is used?

Reply:
It is, it's the bottom border color for active tabs here: 
![Screenshot from 2023-06-22 11-46-38](https://github.com/canonical/canonical.com/assets/17607612/3823aa2d-4717-4724-b3c1-ebc6dcf748c1)

with the old color:
![Screenshot from 2023-06-22 11-46-47](https://github.com/canonical/canonical.com/assets/17607612/c64600d3-cc59-441a-8a43-5e1620a94e78)